### DISCUSSION TITLE
Proposal: Enable branch update suggestions

### DISCUSSION THREAD

**Discussion by bryevdv** [bokeh/bokeh#12443] [Build / CI]
@bokeh/core I'd like to propose turning this option on:

<img width="810" alt="Screen Shot 2022-09-30 at 11 31 40" src="https://user-images.githubusercontent.com/1078448/193334478-134214cc-0665-4649-8bf5-7cd72a40f8f3.png">

In case you push a PR and happen to be behind the base branch, the GH web UI will add a button to update to latest (if possible) right there on the PR page. This seems to be new, I've encountered this in another project and it was convenient. 

Let me know if there are any concerns, if  I don't hear anything I will plan to turn it on next week.

**Comment by mattpap** [comment]
As long as it's only a suggestion, it's perfectly fine to enable.

**Reply by bryevdv** [reply]
Yes it is a button you can push or not push.

**Comment by bryevdv** [comment]
Here is what it looks like in practice:
<img width="1005" alt="Screen Shot 2022-10-01 at 18 38 07" src="https://user-images.githubusercontent.com/1078448/193434230-d1759a3d-c7a0-48cb-bb17-26c0101792f5.png">

I have mixed feelings about merge commit as the default, and I don't think it can be changed at present. But given we squash merge PRs I don't actually think it would matter much in practice.
